src/utils/internal.ts

```diff
@@ -1891,3 +1892,28 @@ export function areSameClaims(
 
   return ClaimType[type] === claim.type;
 }
+
+/**
+ * @hidden
+ */
+export function assertCaAssetValid(id: string): string {
+  if (id.length >= 32) {
+    let guid = id;
+
+    if (!Guid.isGuid(id)) {
```

Comment:
```suggestion
    const uuidRegex = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/
    
     if (!uuidRegex.test(id)) {
```

Reply:
There are multiple open issues regarding this function btw: https://github.com/snico-dev/guid-typescript/issues